html-forms/index.html

```diff
@@ -0,0 +1,80 @@
+<!DOCTYPE html>
+<html>
+  <head>
+    <title>HTML Forms</title>
+  </head>
+  <body>
+
+    <div>
+      <form action="index.html" method="get">
+      <div>
+        <label for="email">Email</label>
+      </div>
+      <div>
+        <input id="email" type="text" name="email" maxlength="30">
+      </div>
+      </form>
+    </div>
+
+    <div>
+      <form action="index.html" method="get">
+      <div>
+        <label for="why is coding cool">Why is coding cool?</label>
```

Comment:
Please do not include spaces in values for `id` or `for`. You can substitute `-` for each space